azurelinuxagent/ga/exthandlers.py

```diff
@@ -390,7 +390,12 @@ def handle_enable(self, ext_handler_i):
         old_ext_handler_i = ext_handler_i.get_installed_ext_handler()
         if old_ext_handler_i is not None and \
            old_ext_handler_i.version_gt(ext_handler_i):
-            raise ExtensionError(u"Downgrade not allowed")
```

Comment:
Can you implement a test case for this?  I would like to capture this behavior, and ensure people know it is defined.